admin-dev/themes/new-theme/js/pages/product/services/combinations-service.js

```diff
@@ -73,7 +73,7 @@ export default class CombinationsService {
    */
   removeCombination(combinationId) {
```

Comment:
```suggestion
  deleteCombination(combinationId) {
```

You forgot to rename this one ^^ which means you probably didn't test this last modification